src/components/BreadCrumb/BreadCrumb.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Breadcrumbs, BreadcrumbItem } from '@nextui-org/react';
+
+export interface BreadCrumbProps {
+  items: { label: string; href?: string }[];
+}
+
+const BreadCrumb: React.FC<BreadCrumbProps> = ({ items }) => {
+  const [currentPage, setCurrentPage] = React.useState(
+    items.length > 0 ? items[items.length - 1].label : '',
+  );
+
+  if (items.length === 0) {
+    return null; // Render nothing if there are no items
+  }
+
+  const handleClick = (itemLabel: string, href?: string) => {
+    setCurrentPage(itemLabel);
```

Comment:
This won't have any impact if the page is redirected to the specified href location.